packages/online-editor/webpack.config.ts

```
@@ -53,6 +54,13 @@ export default async (env: any, argv: any) => {
   ] = getDevDeploymentsArgs();
   const gtmResource = getGtmResource();
 
+  let lastCommitHash = "";
+  try {
+    lastCommitHash = childProcess.execSync("git rev-parse --short HEAD").toString().trim();
+  } catch (e) {
+    console.error(e);
+  }
```

Hmm... here I think you should throw an exception if you can't get this information. We shouldn't be able to build if not inside a versioned directory... Also, you might want to JSON.stringify() inside the try/catch block.